scala-setup.sh

```diff
@@ -32,4 +32,15 @@ SCALA_CLI_BIN_FILE="${TMP_DIR}/scala-cli"
 curl -fLo ${SCALA_CLI_ARCHIVE} $URL
 gzip -d ${SCALA_CLI_ARCHIVE}
 chmod +x ${SCALA_CLI_BIN_FILE}
-mv ${SCALA_CLI_BIN_FILE} /usr/local/bin/scala-cli
\ No newline at end of file
+"${SCALA_CLI_BIN_FILE}" install-home --scala-cli-binary-path ${SCALA_CLI_BIN_FILE}
+rm ${SCALA_CLI_BIN_FILE}
+
+case "$UNAME" in
+    Linux)
+        echo "'~/.profile' has been updated"
```

Comment:
It might update `~/.zprofile` too, we can make this message more generic:
```suggestion
        echo "Profile file(s) updated."
```